Rubgems pubilsh action

After new release tag is added, new gem should be build and published

A new error:

```
  
Run bundle exec rake release
  
rake aborted!
Don't know how to build task 'release' (See the list of available tasks with `rake --tasks`)
/home/runner/work/bddsm/bddsm/vendor/bundle/ruby/3.3.0/gems/rake-13.2.1/exe/rake:27:in `<top (required)>'
/opt/hostedtoolcache/Ruby/3.3.3/x64/bin/bundle:25:in `load'
/opt/hostedtoolcache/Ruby/3.3.3/x64/bin/bundle:25:in `<main>'
(See full trace by running task with --trace)
Error: Process completed with exit code 1.
```

https://github.com/HeavyTechRuby/bddsm/actions/runs/9757234217/job/26929178450

--- a/Rakefile
+++ b/Rakefile
@@ -1,19 +1,2 @@
 require_relative 'lib/bddsm/version'
-
-task default: :build
-
-task :build do
-  system 'gem build bddsm.gemspec'
-end
-
-task install: :build do
-  system "gem install bddsm-#{BDDSM::GEM_VERSION}.gem"
-end
-
-task publish: :build do
-  system "gem push bddsm-#{BDDSM::GEM_VERSION}.gem"
-end
-
-task :clean do
-  system 'rm *.gem'
-end
+require 'bundler/gem_tasks'

